extmod/moductypes.c

```diff
@@ -505,8 +505,7 @@ STATIC mp_obj_t uctypes_struct_attr_op(mp_obj_t self_in, qstr attr, mp_obj_t set
             if (IS_SCALAR_ARRAY(sub) && IS_SCALAR_ARRAY_OF_BYTES(sub)) {
                 return mp_obj_new_bytearray_by_ref(uctypes_struct_agg_size(sub, self->flags, &dummy), self->addr + offset);
             }
-            // Fall thru to return uctypes struct object
-        }
+        }   // Fall through - to return uctypes struct object
```

Comment:
Nitpick but to me it's strange using `-` to split up what is otherwise a complete sentence. Also consistency is key, perhaps also use `//Fall through` (capitalized) for all other plases where you add it

Reply:
yes, me too. it was the best solution that GCC would still recognize I could find though.